<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="troy-PC" name="testTicketWeb.qiChe" tests="5" failures="1" timestamp="8 Nov 2015 09:04:11 GMT" time="29.360" errors="0">
  <testcase name="test000login" time="8.786" classname="testTicketWeb.qiChe"/>
  <testcase name="test003login" time="0.001" classname="testTicketWeb.qiChe"/>
  <testcase name="test004login" time="0.002" classname="testTicketWeb.qiChe"/>
  <testcase name="test005login" time="20.571" classname="testTicketWeb.qiChe">
    <failure type="java.lang.AssertionError" message="Don&#039;t find the 登录输入框 element on the 登录页 page.">
      <![CDATA[java.lang.AssertionError: Don't find the 登录输入框 element on the 登录页 page.
	at org.testng.Assert.fail(Assert.java:94)
	at core.UI.waitDisplay(UI.java:138)
	at core.UI.waitDisplay(UI.java:121)
	at initial.piaoWuWebTest.enterHomePage(piaoWuWebTest.java:13)
	at testTicketWeb.qiChe.test005login(qiChe.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- test005login -->
  <testcase name="test006login" time="0.000" classname="testTicketWeb.qiChe"/>
</testsuite> <!-- testTicketWeb.qiChe -->
